includes/public/layouts/default.php

```diff
@@ -43,7 +43,7 @@ class="wppr-review-container <?php echo( empty( $pros ) ? 'wppr-review-no-pros'
 				<div class="review-wu-left">
 					<div class="review-wu-left-top">
 						<div class="rev-wu-image">
-							<a href="<?php echo esc_url( $image_link ); ?>" <?php echo $lightbox; ?> rel="nofollow"
+							<a class="default-img" href="<?php echo esc_url( $image_link ); ?>" <?php echo $lightbox; ?> rel="nofollow"
```

Comment:
why we need this class?

Reply:
I used this class, because if the image size is too large, AMP will rule out the default class. So, to have a restriction in AMP, I added this class.

here's the demo: 
![default-img](https://user-images.githubusercontent.com/3204558/37301063-91f72b66-264d-11e8-8dab-570272b77cae.gif)

Reply:
@rohittm default image is too generic, we can use `wppr-default-image`

Reply:
Made the changes.